libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.usecase.ts

```diff
@@ -219,6 +219,20 @@ export class UpdateWorkflow {
          */
 
         updatePayload.critical = command.critical;
+
+        this.analyticsService.track(
+          'Workflow critical status changed',
+          command.userId,
+          {
+            _organization: command.organizationId,
+            name: updatePayload.name ?? existingTemplate.name,
+            description:
+              updatePayload.description ?? existingTemplate.description,
+            new_status: command.userPreferences?.all?.readOnly,
+            tags: updatePayload.tags ?? existingTemplate.tags,
+          },
+        );
+
         this.analyticsService.track(
```

Comment:
Should we remove the old analytics event?
@SokratisVidros @LetItRock

Reply:
Yes, you can remove it.